.travis.yml

```diff
@@ -47,7 +47,7 @@ matrix:
       before_install: nvm install 12
     - name: "Python 3.7 on macOS"
       os: osx
-      #osx_image: xcode11
+      osx_image: xcode11
```

Comment:
This should be a separate commit, I think?

Reply:
Did it perhaps come from an incomplete rebase onto or merge from master? It does seem unrelated to the stdout encoding.

Reply:
It was one of the items mentioned at https://github.com/nodejs/node-gyp/pull/1893#pullrequestreview-293495919